### PR TITLE
Publicar Manipulação e transformação de dados com R

### PR DESCRIPTION
**Pull request by DanielAlvesLABDH**
Publicar a tradução da lição "Data Wrangling and Management in R"

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Assign at least one individual or team to "Reviewers"
- [x] Add the appropriate "Label"
- [x] [Ensure the status checks pass](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#4-check-your-pr-status)
- [x] [Check the live preview of your PR on Netlify](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#5-preview-how-your-pr-looks-when-built-into-html)
- [ ] If this PR closes an open issue, add the phrase `Closes #ISSUENUMBER` to the description above

*If you are having difficulty fixing build errors, first consult <https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions> carefully, especially ["Common Build Errors"](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#common-build-errors). Then contact the technical team if you need further help.*

### CODE REVIEW

File: pt/licoes/manipulacao-transformacao-dados-R.md [hunk 1]
```diff
@@ -0,0 +1,422 @@
...
+avatar_alt: Barra de sabão
+doi: 10.46430/phpt0035
+---
+
+{% include toc.html %}
+
+## Requisitos
+Nesta lição consideramos que já possui algum conhecimento da linguagem R. Se ainda não completou a lição [Noções básicas de R com dados tabulares](/pt/licoes/nocoes-basicas-R-dados-tabulares), recomendamos que o faça primeiro. Ter experiência com outras linguagens de programação também pode ser benéfico. Se está buscando por onde começar aprendendo outras linguagens, recomendamos os excelentes tutoriais de Python do *Programming Historian*.
+
+Nota da tradução: o conteúdo da programação utilizado na lição original foi alterado para esta versão em português para que o contexto e os exemplos sejam próximos da realidade da comunidade que fala o idioma. Por conta disso, parte do texto da lição traduzida, bem como os exemplos e as interpretações dos dados são diferente da lição original. No entando, o conteúdo e a estrutura da lição são fidedignos à lição original, como os tipos de dados e as análises desenvolvidas. Mudamos, por exemplo, a comparação entre Mississipi e Virgínia por Brasil e Argentina, mantendo os recursos e procedimentos realizados por Nabeel Siddiqui.
```
Comment (ericbrasiln) [inline pt/licoes/manipulacao-transformacao-dados-R.md]: corrigir "No entando" para "No entanto".

**Review by ericbrasiln — CHANGES_REQUESTED** [review]
Fiz a revisão da lição e executei o código.
Há apenas um pequeno erro de ortografia, que indiquei.

File: pt/licoes/manipulacao-transformacao-dados-R.md (hunk 1 above)
Comment (DanielAlvesLABDH) [reply pt/licoes/manipulacao-transformacao-dados-R.md]: Feito. Obrigado

**Review by ericbrasiln — APPROVED** [review]
Pronto!

**Pull request merged**